src/middleware/navigationMiddleware.ts

```diff
@@ -47,6 +47,21 @@ export const NAVIGATION: Navigation = {
         allowedReferers: [],
         redirectTo: constants.VIEW_USERS_FULL_URL,
         allowedUserRoles: [UserRole.OWNER, UserRole.ADMIN]
+    },
+    [getChangeMemberRoleFullUrl("")]: {
```

Comment:
It would be good to extract this to its own constant so that we don't need to call it with a blank string each time, i.e

```ts
export const CHANGE_MEMBER_ROLE_BASE = `${constants.LANDING_URL}/${constants.EDIT_MEMBER_ROLE_PAGE}`;
export const getChangeMemberRoleFullUrl = (id: string): string => `${CHANGE_MEMBER_ROLE_BASE}/${id}`;
```

```suggestion
    [CHANGE_MEMBER_ROLE_BASE]: {
```

If that makes sense?